releasenotes/notes/tls-inbound-all-protocols.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: security
```

This is necessary to generate release notes with a linked issue or pull request. Otherwise, it will be an additional work see https://github.com/istio/istio/pull/30551
```suggestion
issue:
  - 30590

releaseNotes:
```